content/00.zksync-era/40.tooling/20.hardhat/30.plugins/70.hardhat-zksync-upgradable.md

```diff
@@ -26,266 +39,358 @@ Ensure that you're using the correct version of the plugin for compatibility wit
 
 - Plugin version **<1.6.0** is compatible with `@openzeppelin/contracts-upgradeable` **v4**.
 - Plugin version **≥1.7.0** is compatible with `@openzeppelin/contracts-upgradeable` **v5** (⭐ Recommended)
```

Comment:
Keep this consistent with other compatible parts, so:

- For plugin version **<1.6.0**:
   - Compatible with `@openzeppelin/contracts-upgradeable` **v4**